instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/cache/weaklockfree/AbstractWeakConcurrentMap.java

```diff
@@ -0,0 +1,362 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
```

Comment:
Need license header like

https://github.com/open-telemetry/opentelemetry-java/blob/main/context/src/main/java/io/opentelemetry/context/internal/shaded/WeakConcurrentMap.java#L6

Reply:
I didn't find that license header anywhere, but sync'd entire weaklockfree package with sdk repo (so is now included) 👍